src/couch_jobs/src/couch_jobs_fdb.erl

```diff
@@ -355,6 +357,13 @@ get_activity_vs_and_watch(#{jtx := true} = JTx, Type) ->
     end.
 
 
+get_active_jobs(Type) ->
```

Comment:
Since we are calling it from the outside of couch_jobs let's add this function to couch_jobs module.  There it could look like:

```
get_active_jobs(Tx, Type) ->
    couch_jobs_fdb:tx(couch_jobs_fdb:get_jtx(Tx), fun(JTx) ->
        couch_jobs_fdb:get_active_since(JTx, Type, {versionstamp, 0, 0}})
    end).
```

`Tx` could then be `undefined`, a plain transaction from erldb or a TxDb one from `fabric2_db`